### DISCUSSION TITLE
Text and Spacer Widget

### DISCUSSION THREAD

**Discussion by Meykaefer** [opengisch/QField#4376] [Q&A]
Hi everyone,

With QGIS 3.30, two new widgets were introduced to the 'Attributes Form' - a Text and a Spacer widget. I tried to implement them in a QFIELD project, but I don't think they are supported at this point (this is also what the documentation shows). I know I can achieve similar results with an HTML widget, but I wonder if there are any plans to implement these two widgets. They are easier to use and might be a good and fast way to provide instructions in some data collection forms.

**Comment by jpnunn** [comment]
Hi I tested the text widget and it seems to work in QField on a device in case anyone else is wondering

**Comment by nirvn** [comment]
@Meykaefer , FYI, your discussion thread was migrated to our community platform here (https://community.qfield.org/t/text-and-spacer-widget/426). You'll find some good news in there.
